tests/integration/llm/client.py

```diff
@@ -550,13 +550,13 @@ def get_model_name():
         "max_memory_per_gpu": [22.0],
         "batch_size": [1, 4],
         "seq_length": [256],
-        "tokenizer": "baichuan-inc/Baichuan2-7B-Base"
+        "tokenizer": "TheBloke/Llama-2-13B-fp16"
```

Comment:
we can omit this I'd say - without a tokenizer we'll report word level metrics instead of token level. I'd prefer that instead of using a tokenizer that can produce different results

Reply:
Sounds good, I've removed it for now. I'm unsure how we use the awscurl results today, but we might want to revisit not using the tokenizer for some models.